chaste_codegen/chaste_model.py

```diff
@@ -350,7 +359,7 @@ def _convert_membrane_stimulus_current(self):
         if self._membrane_stimulus_current_orig is None:
             return None, self._stimulus_units
         # get derivative equations
-        d_eqs = self.get_equations_for(self._y_derivatives)
+        d_eqs = self._model.get_equations_for(self._y_derivatives)
```

Comment:
I'd be inclined to add an `optimise=False` argument to `self.get_equations_for` as otherwise it isn't obvious in the call what effect the different method has.

Reply:
I did think of that but wasn't sure it was worth it